webplugin/js/app/media/media-service.js

```diff
@@ -6,6 +6,7 @@ Kommunicate.mediaService = {
         });
     },
     processMicClickedEvent: function () {
+        kmWidgetEvents.eventTracking(eventMapping.onVoiceIconClick);
```

Comment:
what will happen if there is error in user provided function? will it cause problems of further execution?

Reply:
ratifed